outline.md

```diff
@@ -0,0 +1,55 @@
+## Growing an Inner Source Program
+
+45 min
+
+## Short Description
+
+"Inner source is a great idea! Go do inner source!"
+You're the dedicated champion for inner source in your company.
+What do you do now?
+How do you realistically affect the behavior of dozens or hundreds of teams to the point where robust and pervasive inner sourcing is a normal part of the way that engineering is done?
+
+This presentation shares principles, practical experience, and relatable examples on this situation.
+While not claiming all of the answers, it is insightful summary of over a year's experience and work in the area.
+
+## Detailed description
+
+#### Outline
+
+An inner source program is the set of tools, documentation, training, and so forth provided so that any team can participate in inner source effectively as either a maintainer or contributor.
```

Comment:
Thanks for this comment, Max.  I'll try to give attention, care, and iteration to any definition presented so that we have something that we're happy with sticking around for a while.

Looking at both of our definitions as they stand, I feel that they are harmonious.   Differences between them involve one extending the points of the other or bringing up a new point rather than any contradiction.

On the question of a formal organization, I feel that question of how to _implement_ an inner source program is orthogonal to defining what _is_ an inner source program.  Different implementations may work in different context to achieve the resulting program and its effect on engineers.

Reply:
> On the question of a formal organization, I feel that question of how to implement an inner source program is orthogonal to defining what is an inner source program

Yes, I think you are right!